Packages/ClientRuntime/Sources/Networking/Http/Middlewares/LoggerMiddleware.swift

```diff
@@ -35,18 +34,12 @@ public struct LoggerMiddleware<Output: HttpResponseBinding,
             logger.debug("Request: \(input.debugDescriptionWithBody)")
         }
         
-        let response = next.handle(context: context, input: input)
+        let response = try await next.handle(context: context, input: input)
         
-        do {
-            let output = try response.get()
-            if clientLogMode == .response || clientLogMode == .requestAndResponse {
-                logger.debug("Response: \(output.httpResponse.debugDescription)")
-            } else if clientLogMode == .requestAndResponseWithBody || clientLogMode == .responseWithBody {
-                logger.debug("Response: \(output.httpResponse.debugDescriptionWithBody)")
-            }
-            
-        } catch {
-            return response
+        if clientLogMode == .response || clientLogMode == .requestAndResponse {
+            logger.debug("Response: \(response.httpResponse.debugDescription)")
+        } else if clientLogMode == .requestAndResponseWithBody || clientLogMode == .responseWithBody {
+            logger.debug("Response: \(response.httpResponse.debugDescriptionWithBody)")
```

Comment:
We seem to be changing the behavior here in that before we were ignoring any exceptions being thrown and returning the response, but now, if getting the response throws an exception, we no longer return a response.  This might be exactly what we want to do, but not clear on if this is the expected behavior.

Reply:
don't see how we are changing the behavior? we just removed the do catch now that this is a throws function. if line 37 fails it will just throw and it will never get to line 39 exactly as the code was before?

Reply:
my bad, I didn't see the try on line 41 of the old code.